src/pipeline/mod.rs

```diff
@@ -3,7 +3,6 @@ use std::fmt::Display;
 use crate::{pedal, Result};
 use anyhow::anyhow;
 
-#[derive(Debug)]
```

Comment:
What's the effect of removing this? `Pipeline` no longer then (declaratively) implements `Debug`? Who cares if it implements `Debug` - only code that actually tries to debug-log it?

Reply:
Yeah, no real effect right now since `Pipeline` isn't being logged anywhere. I had to remove it because `RingBuffer` doesn't implement `Debug`, so `Pedal` can no longer require `Debug`. (I could have worked around that by manually implementing it for pedal, but we're not even using it so I didn't bother.)